src/main/kotlin/it/pagopa/wallet/config/RedisConfiguration.kt

```diff
@@ -21,15 +21,15 @@ class RedisConfiguration {
     @Bean
     fun npgSessionRedisTemplate(
         redisConnectionFactory: RedisConnectionFactory,
-        @Value("\${wallet.session.ttl}") ttl: Long,
+        @Value("\${wallet.session.ttlSeconds}") ttlSeconds: Long,
     ): NpgSessionsTemplateWrapper {
         val npgSessionRedisTemplate = RedisTemplate<String, NpgSession>()
         npgSessionRedisTemplate.connectionFactory = redisConnectionFactory
         val jackson2JsonRedisSerializer = buildJackson2RedisSerializer(NpgSession::class.java)
         npgSessionRedisTemplate.valueSerializer = jackson2JsonRedisSerializer
         npgSessionRedisTemplate.keySerializer = StringRedisSerializer()
         npgSessionRedisTemplate.afterPropertiesSet()
-        return NpgSessionsTemplateWrapper(npgSessionRedisTemplate, Duration.ofMinutes(ttl))
+        return NpgSessionsTemplateWrapper(npgSessionRedisTemplate, Duration.ofSeconds(ttlSeconds))
```

Comment:
I would propose the use of both duration and time unit. 
We could use `Duration.of(ttl, ChronoUnit.valueOf(timeUnit))` where:
`ttl = WALLET_SESSION_TTL_DURATION`
`timeUnit = WALLET_SESSION_TTL_TIME_UNIT`

With current values it should be something like `Duration.of(1800, ChronoUnit.valueOf("SECONDS"))`

Reply:
IMHO I would not make temporal unit configurable for the following reasons:

1. there would be two system property to configure one single parameter (the timeout)
2. could expose to runtime errors in case of typo in temporal unit name (avoidable with configuration validation)
3. TTL configured in seconds gives the right flexibility to the configuration (and is in the same time unit of the parameter used for [wallet expiration timeout](https://github.com/pagopa/pagopa-wallet-service/blob/de14d2faacee611abd5bc39e4d012ad8bbf4daaf/helm/pay-wallet-values-prod.yaml#L145), I would not see any use case for which configure this parameter in milli or micro seconds for example

What do you think?